resources/js/Components/ComponentsPraktikans/PraktikanCard.jsx

```diff
@@ -0,0 +1,53 @@
+import daskomIcon from "../../../../resources/assets/daskom.svg";
+import CircularImage from "../CircularImage";
+
+export default function CardPraktikan() {
+
+    //data variable
+    let name = "Dhea Aisyah Putri";
+    let nim = "1104223146";
+    let kelas = "TF-46-04";
+    let notel = "081250770473";
+    let email = "dheaaisyahputri09@gmail.com";
+    let major = "S1 Teknik Fisika";
+
+    return (
+        <>
+            <div className="my-auto mx-auto font-poppins">
+                <div className=" bg-softIvory  h-[436px] w-[352.8px] shadow-xl rounded-lg pt-5">
```

Comment:
```suggestion
                <div className="bg-softIvory h-[436px] w-[352.8px] shadow-xl rounded-lg pt-5">
```
please make sure the whitespace is clean here 👌 , other than that, have you make sure that this height and width value would also make this component looks good in a phone, or would that be out of scope?

Reply:
after all pages and components are finished I will make this website responsive and comfortable to view on several types of screen resolutions, but I will not make it in a mobile phone version. supaya praktikannya ga malas buka laptop bang hehehe

Reply:
i see haha, okay sure thats fine